website/orders/api/v1/urls.py

```diff
@@ -0,0 +1,36 @@
+from django.urls import path, register_converter
+
+from orders.api.v1.views import (
+    OrderView,
+    ShiftListCreateAPIView,
+    shift_add_time,
+    shift_add_capacity,
+    shift_scanner,
+    OrderListAPIView,
+    OrderRetrieveUpdateDestroyAPIView,
+    ProductListAPIView,
+    ShiftRetrieveUpdateAPIView,
+    product_search,
+    OrderCreateAPIView,
+)
+from orders.converters import ShiftConverter
+
+register_converter(ShiftConverter, "shift")
+
+urlpatterns = [
+    path("shifts", ShiftListCreateAPIView.as_view(), name="shifts_listcreate"),
+    path("shifts/<int:pk>", ShiftRetrieveUpdateAPIView.as_view(), name="shift_retrieveupdate"),
+    path("shifts/<shift:shift>/orders", OrderListAPIView.as_view(), name="orders_list"),
+    path(
+        "shifts/<shift:shift>/orders/<int:pk>",
+        OrderRetrieveUpdateDestroyAPIView.as_view(),
+        name="orders_retrieveupdatedestroy",
+    ),
```

Comment:
I think it would be nice to add an endpoint /shift/<shift:shift>/orders/<int:pk>/ready to which you can PUT to make the order ready, and idem with paid.

Reply:
This can be done with a PATCH to the endpoint for a specific Order object.